.ci_support/linux_64_c_compiler_version11c_stdlib_version2.17cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11fortran_compiler_version11mpimpichnumpy1.22python3.10.____cpython.yaml

```diff
@@ -0,0 +1,79 @@
+arpack:
+- '3.9'
+bzip2:
+- '1'
+c_compiler:
+- gcc
+c_compiler_version:
+- '11'
+c_stdlib:
+- sysroot
+c_stdlib_version:
+- '2.17'
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- '11.8'
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '11'
+docker_image:
+- quay.io/condaforge/linux-anvil-cuda:11.8
+fftw:
+- '3'
+fortran_compiler:
+- gfortran
+fortran_compiler_version:
+- '11'
+libblas:
+- 3.9 *netlib
+libboost_devel:
+- '1.84'
+liblapack:
+- 3.9 *netlib
+libnetcdf:
+- 4.9.2
+mpi:
+- mpich
+mpich:
+- '4'
+netcdf_fortran:
+- '4.6'
+numpy:
+- '1.22'
+openmpi:
+- '5'
```

Comment:
@mattwthompson I have merged the main to this branch and rerendered the branch. You could see that openmpi is version 5 and zlib is 1